src/Namespace.js

```diff
@@ -56,7 +56,10 @@ export default class Namespace<Value> {
     }
 
     /**
-     * Get the current value of a name.
+     * Get the current value of a name, or `undefined` if there is no value.
+     * Note: Do not use `if (namespace.get(...))` to detect whether a macro
+     * is defined, as the definition may be the empty string which evaluates
+     * to `false` in JavaScript.  Use `if (namespace.get(...) != null)`.
```

Comment:
Can you add a `namespace.has(name)` method that encapsulates this logic?

Reply:
Added, and used in `isDefined`, but I'd hesitate to use it in e.g. `_getExpansion` because it'd double the number of lookups.